ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/InstallPlaywrightTask.cs

```diff
@@ -0,0 +1,13 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks
+{
+    public class InstallPlaywrightTask : GithubTask
```

Comment:
Not expecting this file.  There is another open PR for this component.  Can we remove this file from the PR

Reply:
Yeah, That's because this PR's branche is based on the other one @cjdutoit